docs/compilation/compression.md

```diff
@@ -1,15 +1,17 @@
 # Compression
+This document explains the compression feature in **Concrete** and the performance impact of compression.
```

Comment:
I would replace 

and the performance impact of compression. -> and its performance impact